ROADMAP.md

```diff
@@ -4,6 +4,9 @@
 - [ ] propagate cost in integrator
 - [ ] restore download and compilation of OOQP
 - [ ] split ocp solve into prepare and feedback
+- [ ] stage transition functions for changing model dimensions
+- [ ] fix bghp constraints to have np != 1;
```

Comment:
In general, the p constraints should be also reimplemented to be independent of `h` instead of on top of them, i.e. to be able to have a `p` constraint and an independent `h` constraint on the same stage.

Reply:
I think @zanellia  said he wanted to implement that by using the identity as an outer function in the `p` constraints.
Anyway, this was just to not forget.

Reply:
Now that it is convex-over-nonlinear (instead of just quadratic-over-nonlinear) it is indeed possible, but having the two disjointed would still be the most clean way.

Reply:
And more consistent with the constraints `BGH`, otherwise `H` means two different things in the two constraints types.

Optionally, the current `BGHP` constraint type should be renamed `BGP`, to make clear that there is no stand-alone `H`.